src/main/kotlin/no/nav/dagpenger/inntekt/Api.kt

```diff
@@ -1,13 +1,62 @@
 package no.nav.dagpenger.inntekt
 
-class Api {
+import io.ktor.application.Application
+import io.ktor.application.call
+import io.ktor.application.install
+import io.ktor.features.CallLogging
+import io.ktor.features.ContentNegotiation
+import io.ktor.features.DefaultHeaders
+import io.ktor.gson.gson
+import io.ktor.response.respond
+import io.ktor.routing.get
+import io.ktor.routing.route
+import io.ktor.routing.routing
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+import java.math.BigDecimal
+import java.util.concurrent.TimeUnit
 
-    fun some(): Boolean = true
+class Api {
```

Comment:
I think the plain main method approach like used in : https://github.com/navikt/helse-sparkel/blob/master/src/main/kotlin/no/nav/helse/App.kt#L54 is moar neat than the companion object.

Reply:
oo, cool!